sharktank/sharktank/utils/cli.py

```diff
@@ -22,16 +25,16 @@ def create_parser(
     description: Optional[str] = None,
 ) -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(prog=prog, usage=usage, description=description)
-
     return parser
 
 
 def parse(parser: argparse.ArgumentParser):
     """Parses arguments and does any prescribed global process setup."""
-    return parser.parse_args()
+    args = parser.parse_args()
+    return args
 
 
-def add_gguf_dataset_options(parser: argparse.ArgumentParser):
+def add_input_dataset_options(parser: argparse.ArgumentParser):
     """Adds options to load a GGUF dataset.
 
     Either the `--hf-dataset` or `--gguf-file` argument can be present.
```

Comment:
nit: docstring is out of date (thanks for adding the IRPA argument!)
```suggestion
    Either the `--hf-dataset`, `--gguf-file`, or `--irpa-file` argument can be present.
```